eng/illink.targets

```diff
@@ -24,7 +24,7 @@
     <ILLinkRewritePDBs Condition="'$(ILLinkRewritePDBs)' == '' and '$(DebugSymbols)' != 'false'">true</ILLinkRewritePDBs>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(ILLinkTrimXml)' != ''">
+  <ItemGroup Condition="Exists('$(ILLinkTrimXml)')">
```

Comment:
The exists condition like this just make for hard to debug problems. If one specifies ILLinkTrimXml file, it should better exist.

Reply:
reverted